### PR TITLE
[Feature] TensorRT test tools.

### PR DESCRIPTION
**Pull request by grimoire**
This PR adds TensorRT test tools.

### CODE REVIEW

File: docs/tutorials/onnx2tensorrt.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ python tools/deployment/onnx2tensorrt.py \
     ${MODEL} \
     --trt-file ${TRT_FILE} \
     --shape ${IMAGE_SHAPE} \
-    --workspace-size {WORKSPACE_SIZE} \
+    --max-batch-size ${MAX_BATCH_SIZE} \
+    --workspace-size ${WORKSPACE_SIZE} \
```
Comment (RunningLeon) [inline docs/tutorials/onnx2tensorrt.md:30]: could add `--fp16` as well.

File: docs/tutorials/pytorch2onnx.md [hunk 2]
```diff
@@ -99,6 +99,7 @@ python tools/deployment/test.py \
 
 - `config`: The path of a model config file.
 - `model`: The path of a ONNX model file.
+- `--backend`: Backend for input model to run and should be `onnxruntime` or `tensorrt`.
```
Comment (RunningLeon) [inline docs/tutorials/pytorch2onnx.md:102]: Could add `--backend` after line 89

**Review by RunningLeon — APPROVED** [review]

**Review by RunningLeon — APPROVED** [review]
LGTM

File: tools/deployment/onnx2tensorrt.py [hunk 3]
```diff
@@ -22,6 +24,8 @@ def onnx2tensorrt(onnx_file,
         trt_file (str): Filename of the output TensorRT engine file.
         input_shape (list[int]): Input shape of the model.
             eg [1, 3, 224, 224].
+        max_shape (list[int]): Max input shape of the model.
```
Comment (ycxioooong) [inline tools/deployment/onnx2tensorrt.py:27]: The `max_shape` in docstring does not match any input parameter. You may fix this and specify how max_shape is obtained by setting max_batch_size

**Comment by RunningLeon** [comment]
@ycxioooong Any comments on this PR？

**Comment by ycxioooong** [comment]
LGTM now.

**Pull request merged**